txpipe/selector.py

```diff
@@ -81,10 +82,11 @@ def run(self):
         # various config options
         bands = self.config['bands']
         chunk_rows = self.config['chunk_rows']
+        #print(self.config['input_pz'])
 
-
-        # Build a classifier used to put objects into tomographic bins
-        classifier, features = self.build_tomographic_classifier()        
+        if self.config['input_pz'] == False:
```

Comment:
`if x == True` in python is better expressed as just `if x`.  Similarly for `if x == False` it's better to just do `if not x`